.eslintrc

```diff
@@ -69,6 +69,25 @@
         "browser": false,
         "node": true
       }
+    },
+    {
+      "files": ["packages/components/src/**/*.stories.{ts,tsx}"],
+      "rules": {
+        "no-restricted-imports": [
+          "error", {
+            "patterns": [
+              {
+                "group": [".storybook"],
```

Comment:
using the path-matching pattern didn't work, so I needed to use the [object form](https://github.com/eslint/eslint/blob/04e91b615743cbb25cd9103e7329b056bd1572ad/docs/rules/no-restricted-imports.md#options). I'm guessing that means the former matches from the beginning of the path, which doesn't work for relative imports